model/host/host.go

```diff
@@ -739,3 +744,52 @@ func CountInactiveHostsByProvider() ([]InactiveHostCounts, error) {
 	}
 	return counts, nil
 }
+
+// find all containers
+func FindAllContainers() ([]Host, error) {
+	query := db.Query(bson.M{
+		ContainerIDKey: bson.M{"$exists": true},
+	})
+	hosts, err := Find(query)
+	if err != nil {
+		return nil, errors.Wrap(err, "Error finding containers")
+	}
+
+	return hosts, nil
+}
+
+// find all containers belonging to parent
+func (h *Host) GetContainers() ([]Host, error) {
+	if !h.IsParent {
+		return nil, errors.New("Host is not a parent of any containers")
```

Comment:
Nit: This error message should be "Host is not a parent", since "Host is not a parent of any containers" is ambiguous, suggesting that the host might be a parent, but currently has no children.

Reply:
done